### DISCUSSION TITLE
When will Apps be available on AppSource?

### DISCUSSION THREAD

**Discussion by bwatson23** [microsoft/Templates-for-Power-Platform#92] [General]
It seems there has been a lot of hard work and effort to deploy new solutions, but they don't appear on AppSource?

![image](https://github.com/user-attachments/assets/cc60fe17-0336-4e9d-b1ab-c0eb738e9f09)

![image](https://github.com/user-attachments/assets/f0d44c1e-80a6-4708-af11-2abf8673e4be)

**Comment by mhuguet** [comment]
Tagging @ramanasridhar to help answer.

**Reply by mhuguet** [reply]
The team has published 8 of these and templates for Copilot Studio: https://learn.microsoft.com/en-us/power-platform/enterprise-templates/overview
